<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityWidget</class>
 <widget class="QWidget" name="SecurityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#page_2{background-image: url(:/main/database-background.jpg);}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="SecurityFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#SecurityFrame{background-image: url(:/new/prefix1/res/security-sensor/security-background.png);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_Sensor1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/res/security-sensor/sensor1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>694</x>
        <y>5</y>
        <width>102</width>
        <height>395</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_Door">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>25</y>
         <width>72</width>
         <height>72</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/security-sensor/normal.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Window">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>122</y>
         <width>72</width>
         <height>72</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/security-sensor/normal.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Kitchen">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>219</y>
         <width>72</width>
         <height>72</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/security-sensor/normal.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Carport">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>316</y>
         <width>72</width>
         <height>72</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/security-sensor/normal.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>44</x>
         <y>4</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>门</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>44</x>
         <y>100</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>窗</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>197</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>厨房</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>294</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>车库</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_Sensor4">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>343</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/res/security-sensor/sensor4.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Sensor3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/res/security-sensor/sensor3.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Sensor2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>340</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/res/security-sensor/sensor2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_Sensor1">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>70</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>烟雾探测器</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Sensor4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>红外探测器</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Sensor3">
      <property name="geometry">
       <rect>
        <x>552</x>
        <y>55</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>红外探测器</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Sensor2">
      <property name="geometry">
       <rect>
        <x>575</x>
        <y>345</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>干簧门磁</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Close">
      <property name="geometry">
       <rect>
        <x>695</x>
        <y>427</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/main/button1.png);</string>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>40</y>
       <width>341</width>
       <height>400</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>110</y>
       <width>78</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>正常</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>异常</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_Query">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>110</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/main/button1.png);</string>
     </property>
     <property name="text">
      <string>过滤</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Return">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>390</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/main/button1.png);</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
